Submitted By: DJ Lucas <dj AT linuxfromscratch DOT org>
Date: 2012-06-07
Initial Package Version: 2.2
Upstream Status: Not applicable
Origin: self
Description: excludes known to fail tests from jtreg.

--- icedtea-2.2-orig/test/jtreg/excludelist.jdk.jtx	2012-06-02 04:45:10.000000000 +0000
+++ icedtea-2.2/test/jtreg/excludelist.jdk.jtx	2012-06-04 13:29:32.000000000 +0000
@@ -203,3 +203,137 @@ java/awt/FontClass/CreateFont/bigfont.ht
 # of build process (this class has questionable licence).
 java/util/Locale/LocaleEnhanceTest.java
 
+# NoClassDefFoundError: -classpath is set incorrectly in the testcase
+com/sun/net/httpserver/Test2.java
+com/sun/net/httpserver/bugs/B6341616.java
+java/net/Authenticator/B6870935.java
+java/net/Authenticator/B4722333.java
+java/net/Authenticator/BasicTest.java
+java/net/Authenticator/Deadlock.java
+java/net/Authenticator/B4921848.java
+java/net/Authenticator/B4962064.java
+java/net/Authenticator/BasicTest4.java
+java/net/Authenticator/AuthNPETest.java
+java/net/Authenticator/B4678055.java
+java/net/Authenticator/B4933582.sh
+java/net/Authenticator/B4759514.java
+java/net/Authenticator/BasicTest3.java
+java/net/Authenticator/B4769350.java
+java/nio/channels/SocketChannel/ExceptionTranslation.java
+java/nio/channels/ServerSocketChannel/NonBlockingAccept.java
+java/nio/channels/etc/NetworkChannelTests.java
+java/nio/channels/AsynchronousSocketChannel/Basic.java
+java/nio/channels/AsynchronousServerSocketChannel/Basic.java
+sun/net/www/AuthHeaderTest.java
+sun/net/www/protocol/http/DigestTest.java
+sun/net/www/protocol/http/BasicLongCredentials.java
+sun/net/www/protocol/http/NTLMTest.java
+sun/security/ssl/sun/net/www/protocol/https/HttpsURLConnection/HttpsProxyStackOverflow.java
+sun/security/ssl/sun/net/www/protocol/https/HttpsURLConnection/PostThruProxyWithAuth.java
+sun/security/krb5/auto/HttpNegotiateServer.java
+
+# Win32 failure only, exits 0 unexpectedly:
+java/awt/Choice/ChoiceMouseWheelTest/ChoiceMouseWheelTest.java
+
+# robot failure (window not anchored in TWM):
+java/awt/EventDispatchThread/LoopRobustness/LoopRobustness.java
+java/awt/Focus/CloseDialogActivateOwnerTest/CloseDialogActivateOwnerTest.java
+
+# Xorg behavior change, test successfully fails to render invalid screen, but exit is unexpected
+java/awt/GraphicsDevice/CloneConfigsTest.java
+
+# Tests fail unexpectedly if no printers defined
+java/awt/print/PrinterJob/PrtException.java
+javax/print/attribute/AttributeTest.java
+javax/print/attribute/GetCopiesSupported.java
+javax/print/attribute/SidesPageRangesTest.java
+javax/print/attribute/SupportedPrintableAreas.java
+javax/print/CheckDupFlavor.java
+
+# New Xorg or evdev? Both fail on simulated button 9 press
+java/awt/Robot/AcceptExtraMouseButtons/AcceptExtraMouseButtons.java
+java/awt/Toolkit/ToolkitPropertyTest/ToolkitPropertyTest_Enable.java
+
+# This test looks to be WM dependent, window was disposed causing error
+java/awt/Window/Grab/GrabTest.java
+# Same here: Transparent frame is not transparent!
+javax/swing/JComponent/6683775/bug6683775.java
+
+# Timed out (possibly -samevm issue)
+java/awt/Window/WindowType/WindowType.java
+java/awt/WMSpecificTests/Mutter/MutterMaximizeTest.java
+java/beans/EventHandler/Test6179222.java
+java/beans/EventHandler/Test6277266.java
+java/beans/EventHandler/Test6788531.java
+
+# Test arguments invalid
+java/lang/invoke/ClassValueTest.java
+java/lang/invoke/InvokeGenericTest.java
+java/lang/invoke/JavaDocExamplesTest.java
+java/lang/invoke/MethodHandlesTest.java
+java/lang/invoke/MethodTypeTest.java
+java/lang/invoke/PermuteArgsTest.java
+java/lang/invoke/RicochetTest.java
+
+# Trust store not found (invalid argument)
+java/net/URL/TestHttps.java
+
+## Bad tests (I think, reason given before each)
+# Unexpected file type in tree
+java/nio/file/Files/probeContentType/Basic.java
+# Test passes then fails due to "exception: no exception"?!?!
+java/security/KeyPairGenerator/Failover.java
+# new/old jpeg library?
+javax/imageio/plugins/shared/BitDepth.java
+# Access denied error
+javax/management/openmbean/TabularDataOrderTest.java
+javax/swing/JFileChooser/6738668/bug6738668.java
+# Using newer version than expected (1.8.5 when 1.8 is expected)
+javax/script/VersionTest.java
+# Tests cause infinite loop (1.8.5 changes?)
+javax/swing/JEditorPane/6917744/bug6917744.java
+javax/swing/JInternalFrame/Test6325652.java
+javax/swing/JInternalFrame/Test6505027.java
+javax/swing/JInternalFrame/Test6802868.java
+javax/swing/JLabel/6596966/bug6596966.java
+javax/swing/JLabel/7004134/bug7004134.java
+javax/swing/JLayer/6824395/bug6824395.java
+javax/swing/JList/6462008/bug6462008.java
+javax/swing/JOptionPane/7138665/bug7138665.java
+javax/swing/JPopupMenu/4966112/bug4966112.java
+javax/swing/JPopupMenu/6495920/bug6495920.java
+javax/swing/JPopupMenu/6827786/bug6827786.java
+javax/swing/JPopupMenu/6987844/bug6987844.java
+javax/swing/JScrollBar/4708809/bug4708809.java
+javax/swing/JScrollBar/4865918/bug4865918.java
+javax/swing/JScrollBar/6542335/bug6542335.java
+javax/swing/JScrollPane/Test6526631.java
+javax/swing/JSlider/6348946/bug6348946.java
+javax/swing/JSlider/6848475/bug6848475.java
+javax/swing/JSplitPane/4885629/bug4885629.java
+javax/swing/JTabbedPane/4624207/bug4624207.java
+javax/swing/JTable/6263446/bug6263446.java
+javax/swing/JTable/6777378/bug6777378.java
+javax/swing/JTableHeader/6884066/bug6884066.java
+javax/swing/JTableHeader/6889007/bug6889007.java
+javax/swing/JTextArea/4697612/bug4697612.java
+javax/swing/JTree/6263446/bug6263446.java
+javax/swing/JTree/6505523/bug6505523.java
+javax/swing/JViewport/7107099/bug7107099.java
+javax/swing/plaf/basic/BasicHTML/4251579/bug4251579.java
+javax/swing/PopupFactory/6276087/NonOpaquePopupMenuTest.java
+javax/swing/Security/6657138/ComponentTest.java
+javax/swing/SwingUtilities/4917669/bug4917669.java
+javax/swing/text/CSSBorder/6796710/bug6796710.java
+javax/swing/text/DefaultEditorKit/4278839/bug4278839.java
+
+# Still using LCMS1 profile (LCMS2 tests need backported from java 8 tree)
+sun/java2d/cmm/ColorConvertOp/MTSafetyTest.java
+sun/java2d/cmm/ProfileOp/ReadWriteProfileTest.java
+
+# Invalid test case of initializer (constant)
+sun/net/idn/TestStringPrep.java
+
+# Fails to compile
+sun/security/ec/TestEC.java
+
--- icedtea-2.2-orig/test/jtreg/excludelist.langtools.jtx	2012-06-02 04:45:10.000000000 +0000
+++ icedtea-2.2/test/jtreg/excludelist.langtools.jtx	2012-06-02 04:46:29.000000000 +0000
@@ -27,3 +27,7 @@ tools/javap/ListTest.java
 
 # `main' threw exception: java.lang.Error: 64/256 test cases passed
 tools/javap/OptionTest.java
+
+# Fails to compile
+tools/javac/processing/6499119/ClassProcessor.java
+
